packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/utils/useEditorSearchExtension.ts

```diff
@@ -73,15 +77,15 @@ export default function useEditorSearch(CodeMirror: any) {
 				// If we run out of matches then just highlight the final match
 				break;
 			}
-			match = cursor.pos;
+			match = { from: cursor.from(), to: cursor.to() };
 		}
 
 		if (match) {
 			if (scrollTo) {
 				if (withSelection) {
 					cm.setSelection(match.from, match.to);
 				} else {
-					cm.scrollTo(match);
+					cm.scrollIntoView(match);
```

Comment:
According to the CodeMirror documentation, [`.scrollTo` takes two arguments](https://codemirror.net/5/doc/manual.html#scrollTo), while [`scrollIntoView` can take an object with the same type as `match`](https://codemirror.net/5/doc/manual.html#scrollIntoView).